.github/CODEOWNERS

```diff
@@ -18,6 +18,8 @@
 #### Build stuff
 scripts/ @dzearing @ecraig12345 @xugao
 scripts/babel/ @layershifter @miroslavstastny
+scripts/create-package/ @dzearing @ecraig12345 @joschemd @behowell
+scripts/create-component/ @joschemd @behowell
```

Comment:
Any opinions on who to include here? 
Starting with @joschemd @behowell (and @dzearing @ecraig12345 for create-package)
But since it relates to convergence, we might also want to add one or more of @miroslavstastny @layershifter @ling1726 @JustSlone 

Please reply if you think you should be an owner too :)

Reply:
Feel free to add me there 👍

Reply:
Fine with me

Reply:
@ling1726 Do you mean it's fine to not include you? Or would you like to be added as an owner?

Reply:
You should add me, then I didn't have say 'I have the power' out loud incorrectly 😉